readme.md

```diff
@@ -96,7 +96,7 @@ This plugin allows you to interact with Bluetooth LE devices on Android, iOS, an
 
 * Cordova 5.0.0 or higher
 * Android 4.3 or higher, Android Cordova library 5.0.0 or higher, target Android API 23 or higher
-* iOS 7 or higher
+* iOS 10 or higher
```

Comment:
writeQ says it requires iOS11. Should this be iOS11+, or is it just the writeQ function?

Reply:
I actually failed to specify it, only `writeQ` with `type=noResponse` **requires** iOS11 because of the callback introduced in that version otherwise it will run on iOS7+ i believe.